test/helpers.go

```diff
@@ -0,0 +1,19 @@
+package test
+
+import (
+	"encoding/json"
+	"net/http"
+
+	. "github.com/onsi/gomega"
```

Comment:
Does this `.` mean something in specific or it is a typo?

Reply:
It has a special meaning, normally you need to prefix the package name to access functions and variables in other packages, a *dot* import allows you to access those members w/o the package name. This dot import allows us to use `Expect()` directly, otherwise we would have to do `gomega.Expect()`.

It' recommended not to do dot imports, but it's not an error, and in this case it really improves readability.